app/src/main/java/cm/aptoide/pt/timeline/TimelineAnalytics.java

```diff
@@ -655,6 +679,11 @@ public void sendCommentCompleted(boolean success) {
 
   public void sendFabClicked() {
     HashMap<String, Object> data = new HashMap<>();
+    final String previousContext = navigationTracker.getPreviousViewName();
+    final String store = navigationTracker.getPreviousScreen().getStore();
```

Comment:
getPreviousScreen might return null if screen history size is < 2, throwing an NPE